docs/src/user-guide/guides-admin-tools.md

```diff
@@ -1,110 +1,139 @@
-# Using the admin tools
+# Admin tools
 
-CLP provides a set of admin scripts that allow user to manage logs compressed by CLP, located
-under `sbin/admin-tools/`.
-Currently, CLP supports `archive-mananger.sh` and `dataset-manager.sh`, which manage compressed 
-logs at the level of archives and datasets.
+CLP provides a set of scripts that allow system administrators to manage their compressed logs. The
+scripts are located in the `sbin/admin-tools/` directory within the CLP package.
+
+Available scripts:
+
+* [archive-manager.sh](#archive-managersh) for listing and deleting archives.
+* [dataset-manager.sh](#dataset-managersh) for listing and deleting datasets (only for `clp-json`).
 
 :::{note}
 The admin scripts can only be used after CLP starts. For help using CLP, see the
 [clp-json quick start](quick-start/clp-json.md) or [clp-text quick start](quick-start/clp-text.md)
-pages for details.
+for details.
 :::
 
-:::{caution}
-When running the admin scripts, users must not compress or search logs with CLP.  
+:::{warning}
+The admin scripts **should not** be run while users are compressing or searching logs with CLP.
 Doing so may result in undefined behavior.
```

Comment:
I feel it would flow more nature if we say it the other way round, i.e., user should not do <something else> when they run the admin script. but eventually both works.